### ISSUE TITLE
npm run test fails with vite error

### ISSUE THREAD

**Issue by Renpona** [yeemachine/kalidokit#21]
When running npm run test or npm run dev to start up the demo projects in the docs folder, vite fails to start the dev server, claiming a missing dependency in the Live2D folder.

Despite the error claiming that script.js could not be resolved, I've verified that the script.js file is present in docs/live2d, and I haven't touched it or index.html in any way. The only change I've made so far is removing the pre-build script from package.json to workaround #19 

Platform: Windows 10

Steps to reproduce:
1) Clone Kalidokit repository
2) Remove prebuild script from package.json
3) Run "npm install"
4) Run "npm run test"

Full error output:
```
C:\Projects\Programming\kalidokit>npm run test

> kalidokit@1.1.2 test C:\Projects\Programming\kalidokit
> npm run build && npm run dev


> kalidokit@1.1.2 build C:\Projects\Programming\kalidokit
> vite build && npm run build:lib

vite v2.6.0 building for production...
Unknown input options: exports. Allowed options: acorn, acornInjectPlugins, cache, context, experimentalCacheExpiry, external, inlineDynamicImports, input, makeAbsoluteExternalsRelative, manualChunks, maxParallelFileReads, moduleContext, 
onwarn, perf, plugins, preserveEntrySignatures, preserveModules, preserveSymlinks, shimMissingExports, strictDeprecations, treeshake, watch
✓ 13 modules transformed.
dist/kalidokit.es.js   31.87 KiB / gzip: 6.99 KiB
dist/kalidokit.umd.js   16.10 KiB / gzip: 5.38 KiB

> kalidokit@1.1.2 build:lib C:\Projects\Programming\kalidokit
> tsc -p .


> kalidokit@1.1.2 dev C:\Projects\Programming\kalidokit
> vite docs


  vite v2.6.0 dev server running at:

error when starting dev server:
Error: The following dependencies are imported but could not be resolved:

  script.js (imported by C:/Projects/Programming/kalidokit/docs/live2d/index.html)

Are they installed?
    at optimizeDeps (C:\Projects\Programming\kalidokit\node_modules\vite\dist\node\chunks\dep-a8e01c9f.js:71146:15)     
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async runOptimize (C:\Projects\Programming\kalidokit\node_modules\vite\dist\node\chunks\dep-a8e01c9f.js:74997:48)
    at async Server.httpServer.listen (C:\Projects\Programming\kalidokit\node_modules\vite\dist\node\chunks\dep-a8e01c9f.js:75013:21)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! kalidokit@1.1.2 dev: `vite docs`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the kalidokit@1.1.2 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by yeemachine** [comment]
Thanks for the report. Was a url directory issue. Should be fixed with the latest commit a564c5d

**Comment by Renpona** [comment]
Confirmed, the fix works!
